consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/pdl/PdlOppslagServiceImpl.kt

```diff
@@ -1,89 +1,100 @@
 package no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.pdl
 
-import com.google.gson.GsonBuilder
+import com.expediagroup.graphql.client.GraphQLClient
+import io.ktor.client.HttpClient
+import io.ktor.client.features.HttpClientFeature
+import io.ktor.client.request.HttpRequestPipeline
+import io.ktor.client.request.header
+import io.ktor.util.AttributeKey
+import kotlinx.coroutines.runBlocking
 import no.nav.common.auth.SsoToken
 import no.nav.common.auth.SubjectHandler
 import no.nav.common.oidc.SystemUserTokenProvider
 import no.nav.log.MDCConstants
-import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.pdl.PdlPersonResponse
-import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.pdl.PdlRequest
-import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.pdl.Variables
-import no.nav.sbl.dialogarena.modiabrukerdialog.api.service.pdl.PdlOppslagService
+import no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.pdl.generated.HentNavn
+import no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.pdl.generated.HentPerson
 import no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.unleash.strategier.ByEnvironmentStrategy.ENVIRONMENT_PROPERTY
-import no.nav.sbl.dialogarena.modiabrukerdialog.consumer.util.RestConstants.*
-import no.nav.sbl.rest.RestUtils
+import no.nav.sbl.dialogarena.modiabrukerdialog.consumer.util.RestConstants
 import no.nav.sbl.util.EnvironmentUtils
 import org.slf4j.LoggerFactory
 import org.slf4j.MDC
+import java.net.URL
 import java.util.*
 import javax.inject.Inject
-import javax.ws.rs.client.Entity
-import javax.ws.rs.core.HttpHeaders.AUTHORIZATION
+import javax.ws.rs.core.HttpHeaders
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.CoroutineContext
 
+class CoroutineHeaderFeature {
```

Comment:
Yeah, dette er en super-hack for å få sendt med auth-header til pdl.
Har opprettet issue til dem for å høre om det finnes en bedre måte; https://github.com/ExpediaGroup/graphql-kotlin/issues/764